src/entity/model/ArrowSign.c

```diff
@@ -0,0 +1,89 @@
+#include "common.h"
```

Comment:
Need to have a discussion about how to make some kind of splat segment for these model files, they're arguably assets (embedded Vtx, images, Gfx)

Reply:
splat can extract the individual display lists and vtx currently, but I'm not sure if it's worth it. none of this looks very asset-y to me other than the images, which area already handled. but we can def talk about it